### PR TITLE
Adding composer and bower dependency definitions.

### PR DESCRIPTION
**Pull request by grasmash**

### CODE REVIEW

**Comment by grasmash** [comment]
This is a work in progress. Still have not confirmed working functionality. Likely need to do some work with autoloader.

**Comment by grasmash** [comment]
So, there are some interesting consequences of using Bower for front end dependency management:
- Build process is more lightweight. For dependencies, we typically pull in only the "dist" files rather than the entire library.
- The library paths change. We can keep things in /libraries, but the subdirectories change depending on the dependency's "main" value in its bower.json.

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "drupal/page_manager": "8.1.x-dev#3f15c87fec006e78954bb3e984cb80afeea4f5b2",
+    "drupal/acquia_connector": "8.1.x-dev#302fc469eb860892fc90403b5d758eb50a715f77",
+    "drupal/metatag": "8.1.0-beta3",
+    "drupal/core": "8.*"
+  },
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "extra": {
+    "patches": {
+      "drupal/ctools": {
```
Comment (danepowell) [inline composer.json:33]: doesn't media_entity_twitter also have a patch? https://github.com/acquia/lightning/blob/8.x-1.x/drupal-org.make#L33

**Comment by balsama** [comment]
What do you think about this @phenaproxima ? Could we update our prepmake script to automatically generate the composer file too?

**Comment by grasmash** [comment]
@balsama I've created a workaround for this issue on Bolt, so it's not as urgent. I've linked to a number of solutions in progress in #94, perhaps we should see if those move forward?

**Pull request closed without merging**